R/process_data.R

```diff
@@ -129,11 +133,6 @@ process_scenario_data <- function(data, start_year, end_year, sectors, technolog
                                   scenario_geography_filter, scenarios_filter) {
   data_processed <- data %>%
     wrangle_scenario_data(start_year = start_year, end_year = end_year) %>%
-    dplyr::filter(
```

Comment:
is this taken care of when reading the data or why should we remove these?

Reply:
they are duplicates to lines 137 ff

Reply:
oh wow yes..